ocs_ci/ocs/resources/pod.py

```diff
@@ -893,6 +893,26 @@ def get_noobaa_operator_pod(
     return noobaa_operator_pod
 
 
+def get_noobaa_db_pod():
+    """
+    Get noobaa db pod obj
+    Returns:
+        Pod object: Noobaa db pod object
+    """
+    if version.get_semantic_ocs_version_from_config() > version.VERSION_4_6:
```

Comment:
Can't we skip the check and just use `constants.NOOBAA_DB_LABEL_47_AND_ABOVE` since all our supported branches are for versions >= 4.10?